vm/nova/vm_test.go

```diff
@@ -6072,6 +6768,8 @@ func TestTxSemanticMana(t *testing.T) {
 				wantErr: iotago.ErrManaOverflow,
 			}
 		}(),
+
+		// fail - mana overflow on the output side sum
```

Comment:
Maybe you should make a VS Code plugin that generates these as inlay hints :D.

Reply:
It would be nice if the code is readable without fancy plugins :P